system-test/datastore.ts

```diff
@@ -17,20 +17,33 @@ import {readFileSync} from 'fs';
 import * as path from 'path';
 import {after, before, describe, it} from 'mocha';
 import * as yaml from 'js-yaml';
-import {Datastore, Index} from '../src';
+import {Datastore, DatastoreOptions, Index} from '../src';
 import {google} from '../protos/protos';
 import {Storage} from '@google-cloud/storage';
 import {AggregateField} from '../src/aggregate';
 import {PropertyFilter, and, or} from '../src/filter';
 import {entity} from '../src/entity';
 import KEY_SYMBOL = entity.KEY_SYMBOL;
 
+const async = require('async');
+
+const SECOND_DATABASE_ID = 'multidb-test';
+
+async.each(
+    [
+      {
+        namespace: `${Date.now()}`,
+      },
+      {
+        databaseId: SECOND_DATABASE_ID,
+        namespace: `${Date.now()}`,
```

Comment:
nit - slightly update this so that we can easily distinguish between the default namespace and the secondary db namespace; maybe something like `second-db-[date]`

Reply:
Done